networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-infoblox.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/external_dns_operator/nw-creating-dns-records-on-infoblox.adoc
+
+:_content-type: PROCEDURE
+[id="nw-control-dns-records-public-dns-zone-infoblox_{context}"]
+= Creating DNS records on an public DNS zone for Infoblox by using Red Hat External DNS Operator
+
+You can create DNS records on a public DNS zone for Infoblox by using the Red Hat External DNS Operator.
+
+.Procedure
+
+. Create a `secret` with Infoblox credentials:
++
+----
+$ oc -n external-dns-operator create secret generic infoblox-credentials --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=${INFOBLOX_USERNAME} --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=${INFOBLOX_PASSWORD}
+----
+
+. Get the `routes` to check your cluster domain:
++
+[source,terminal]
+----
+$ oc get routes --all-namespaces | grep console
+----
++
+.Example Output
+----
+openshift-console          console             console-openshift-console.apps.aws.devcluster.openshift.com                       console             https   reencrypt/Redirect     None
+openshift-console          downloads           downloads-openshift-console.apps.aws.devcluster.openshift.com                     downloads           http    edge/Redirect          None
+----
+
+.From the UI
+
+. Add an authoritative DNS zone for your cluster domain to Infoblox using the Infoblox web console.
+
+.. Click *Data Management* -> *DNS*.
+.. Click *Add* -> *Zone*
+.. Select *Authoritative Zone* -> *Forward Mapping*.
+.. Enter the cluster domain as zone name (e.g. `aws.devcluster.openshift.com`).
+.. Click *+*.
+.. Click *Save*.
+.. Click *Close*.
+
+.From the CLI
+
+. Create an `ExternalDNS` CR as follows:
++
+[source,yaml]
+----
+apiVersion: externaldns.olm.openshift.io/v1beta1
+kind: ExternalDNS
+metadata:
+  name: sample-infoblox
+spec:
+  provider:
+    type: Infoblox
+    infoblox:
+      credentials:
+        name: infoblox-credentials
+      gridHost: ${INFOBLOX_GRID_PUBLIC_IP}
+      wapiPort: 443
+      wapiVersion: "2.3.1"
+  domains:
+  - filterType: Include
+    matchType: Exact
+    name: aws.devcluster.openshift.com
+  source:
+    type: OpenShiftRoute
+    openshiftRouteOptions:
+      routerName: default
+----
+
+. Check the DNS records created for `console` routes:
++
+[source,terminal]
+----
+$ dig @${INFOBLOX_GRID_PUBLIC_IP} $(oc -n openshift-console get route console --template='{{range .status.ingress}}{{if eq "default" .routerName}}{{.host}}{{end}}{{end}}') +short
+----
++
+.Example Output
+----
+router-default.apps.aws.devcluster.openshift.com
+----
```

Comment:
Just to match the other providers, let's check using the Infoblox web UI.
It's in `Data Management -> DNS -> Zones`, then the zone needs to be choosen and the records are checked.
![image](https://user-images.githubusercontent.com/18031474/170014428-0609685e-98a6-41e7-a5ef-967811b668e6.png)

Reply:
Ack'ed.